### PR TITLE
Update javascript document strings (jsdocs)

### PR DESCRIPTION
**Pull request by vxsx**
adds missing comments, fixes existing comments

in some places it made sense to rename methods for consistency

### CODE REVIEW

File: cms/static/cms/js/modules/cms.structureboard.js [hunk 1]
```diff
@@ -325,7 +371,12 @@
                 }
             },
 
-            /*
+            /**
+             * Resizes the placeholder to fit their placement
+             * and the structure board.
+
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.structureboard.js]: why the space?

**Comment by FinalAngel** [comment]
great work, wanted to do this in the airplane :dancer: 
LGTM!

**Comment by evildmp** [comment]
Could we have a slightly more meaningful PR title please?

**Comment by FinalAngel** [comment]
sure @evildmp

**Pull request merged**